mm2src/lp_native_dex.rs

```diff
@@ -52,10 +52,9 @@ cfg_native! {
 }
 
 #[path = "lp_init/init_context.rs"] mod init_context;
-#[path = "lp_init/mm_init_task.rs"] mod mm_init_task;
-#[path = "lp_init/rpc_command.rs"] pub mod rpc_command;
-
-use mm_init_task::MmInitTask;
+#[path = "lp_init/init_hw.rs"] pub mod init_hw;
+// #[path = "lp_init/mm_init_task.rs"] mod mm_init_task;
```

Comment:
Commented code?

Reply:
I commented out this line because we don't allow the user to initialize mm2 with a Hardware Wallet anymore. I left `mm_init_task.rs` in the repo in case we return this ability.
Do you think I should remove all commented lines, files?

Reply:
Yes, I think so. We will still have these in the git history if we need them.

Reply:
Done